careers/devops.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: DevOps Engineer
+summary: The Haskell Foundation (HF) seeks a passionate DevOps professional to support GHC, related Haskell projects and tooling, and the HF itself.
+postedOn: 2022-01-12
+status: Open
+---
+
+# Description
+
+The [Haskell Foundation](https://haskell.foundation) is [looking for a DevOps engineer](https://github.com/haskellfoundation/tech-proposals/blob/main/proposals/accepted/003-ghc-ops.md) to take the Haskell ecosystem, primarily the [Glasgow Haskell Compiler](https://gitlab.haskell.org/ghc/ghc) (GHC, the Haskell compiler), to the next level of stability, reliability, and performance.
+
+This role spans many Haskell ecosystem projects to homogenize and improve the infrastructure used to build GHC and other core open source projects whose maintainers are interested, for instance HLS, Cabal, Stack, GHCup, core libraries, and more.
+
+The successful candidate will be in the middle of it all, collaborating with developers at the forefront of making Haskell a groundbreaking, influential, top-tier programming language. The individual will improve existing infrastructure, work on creating a shared [CI](https://gitlab.haskell.org/ghc/ghc/-/tree/master/.gitlab) and performance benchmarking system across the Haskell ecosystem, shortened CI turnaround times, and improved ability to debug failures.
+
+# Responsibilities
+
+* Maintain CI (continuous integration) infrastructure for GHC (the Glasgow Haskell Compiler, the leading compiler for the Haskell language), as hosted on GHC's GitLab instance.
+* Work with the GHC team to improve CI efficiency, reliability, and resource utilization.
+* Design and implement expanding the CI infrastructure to include key components of the Haskell ecosystem.
+* Support other Haskell libraries and tools, including HLS (Haskell Language Server).
+* Support OS packaging of GHC and tools.
```

Comment:
Say a bit more?   We are talking about (a) CI and (b) OS packaging, right?  For both GHC and other tools.

Reply:
```suggestion
* Support operating-system-dependent packaging of GHC and related tools.
```